Makefile

```diff
@@ -0,0 +1,22 @@
+
+
+
+MKFILE_DIR := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
+CPP_DIR := $(MKFILE_DIR)timemachine/cpp/
+INSTALL_PREFIX := $(MKFILE_DIR)timemachine/
+
+NPROCS = `nproc`
+
+CUDA_ARCH := "sm_70"
+
+.PHONY: build
+build:
+	mkdir -p $(CPP_DIR)build/ && cd $(CPP_DIR)build/ &&  \
+	cmake -DCUDA_ARCH=$(CUDA_ARCH) -DCMAKE_INSTALL_PREFIX=$(INSTALL_PREFIX) ../ && \
```

Comment:
Would it be possible to enable setting CUDA_ARCH from the commandline or as a env variable somehow?